VERSION

```diff
@@ -1 +1 @@
-1.0.2
+1.0.3
```

Comment:
It's not really adding anything new, just tweaking some settings. I'd say it's very similar to renaming the nginx log group